### ISSUE TITLE
The aria-labelledby attribute must point to IDs of elements in the same document.

### ISSUE THREAD

**Issue by christhepianist** [samvera/hyku#3050]
vpat.hykudemo.org
Version 7.1

### Descriptive summary, including screenshots, if any.
This error was the result of  SortSite accessibility test. This is the information we were provided:
- Guidelines: WCAG 2.1 A 1.3.1 ARIA 1.1  
- Description: The aria-labelledby attribute must point to IDs of elements in the same document. These IDs weren't found: slideshow-modal-label in the URLs below:
https://vpat.hykudemo.org/collections/23615d2a-ac7e-499f-ab76-3bf20bce80b9?locale=de&view=slideshow
https://vpat.hykudemo.org/collections/23615d2a-ac7e-499f-ab76-3bf20bce80b9?locale=es&view=slideshow
https://vpat.hykudemo.org/collections/23615d2a-ac7e-499f-ab76-3bf20bce80b9?locale=fr&view=slideshow
https://vpat.hykudemo.org/collections/23615d2a-ac7e-499f-ab76-3bf20bce80b9?locale=it&view=slideshow
https://vpat.hykudemo.org/collections/23615d2a-ac7e-499f-ab76-3bf20bce80b9?locale=pt-BR&view=slideshow
https://vpat.hykudemo.org/collections/23615d2a-ac7e-499f-ab76-3bf20bce80b9?locale=zh&view=slideshow
https://vpat.hykudemo.org/collections/23615d2a-ac7e-499f-ab76-3bf20bce80b9?locale=en&view=slideshow

### Rationale
The value used in aria-labelledby must use an ID within the document. 
https://www.w3.org/TR/2008/REC-WCAG20-20081211/#content-structure-separation-programmatic
https://www.w3.org/TR/wai-aria-1.1/#aria-labelledby

### Expected behavior
Values used in the aria-labelledby property must be in the HTML document.

### Actual behavior
If not used, this property/value should be removed.

Steps to reproduce the behavior
Run the SortSite accessibility test again. 

Related work (other tickets, possibly the same issue in Hyrax)
